manifests/kubeless.jsonnet

```diff
@@ -1,6 +1,6 @@
 local kube = import "kube.libsonnet";
 
-local kubeless = import "vendor/kubeless/kubeless-rbac.jsonnet";
+local kubeless = import "vendor/kubeless/kubeless.jsonnet";
```

Comment:
is RBAC now the default?

Reply:
yes, the files has been renamed to use RBAC by default